run_dir/design/reads_total.html

```diff
@@ -46,23 +46,34 @@ <h3 id="summary_table_section">Summary of Selected Cluster Counts</h3>
         {% set counter = 0 %}
         <div class="row">
         {% for sample in readsdata %}
-            {% if counter%3==0 %}</div><hr><div class="row">{% end %}
-            <div id="{{ sample }}" class="col-md-4 sample_table" style="padding-top:15px;">
+            {% if counter%2==0 %}</div><hr><div class="row">{% end %}
+            <div id="{{ sample }}" class="col-md-6 sample_table" style="padding-top:15px;">
                 <table id="table_{{ sample }}"class="table reads_table">
                     <tr>
                         <th><a href="/project/{{ sample.split('_')[0] }}">{{ sample }}</a></th>
+                        <th>% > q30</th>
```

Comment:
HTML entities for `>` here. Also using non-breaking spaces will stop the header from wrapping onto a new line which look better I think.
```html
<th>%&nbsp;&gt;&nbsp;q30</th>
```